BoldGrid/boldgrid-backup
# Add specificity to several nonces. Fixes #332

To test:
1. Set [the threshold for the rating prompt to show](https://github.com/BoldGrid/boldgrid-backup/blob/master/includes/config/config.rating-prompt.php#L151) after 1 backup
1. Make sure the rating prompt shows and (1) make sure the archive browser works, (2) the database browser works, and (3) you can successfully click one of the "upload" buttons next to a remote storage provider.
